--- testAnonymousEngineRestore.cc	2022-12-20 18:01:49
+++ testAnonymousEngineRestore_mod.cc	2024-10-15 18:58:59
@@ -121,7 +121,7 @@
 void anonymousRestore1(int n, std::vector<double> & v) {
   output << "Anonymous restore for " << E::engineName() << "\n";
   E e(12349876);				    
-  double __attribute__ ((unused)) r = 0;
+  double r = 0;
   for (int i=0; i<n; i++) r += e.flat();	    
   std::ofstream os("anonymous.save");		    
   os << e;					    
@@ -141,7 +141,7 @@
   std::vector<double> nonRand = aSequence(500);
   NonRandomEngine e; 
   e.setRandomSequence(&nonRand[0], (int)nonRand.size());
-  double __attribute__ ((unused)) r = 0;
+  double r = 0;
   for (int i=0; i<n; i++) r += e.flat();
   std::ofstream os("anonymous.save");
   os << e;
